privcount/traffic_model.py

```diff
@@ -276,8 +281,10 @@ def _get_inter_packet_delays(self, strm_start_ts, byte_events):
             delay = max(long(0), long(micros))
 
             # create a packet delay for each packet
-            while bw_bytes > 0:
+            split_packet_count = 0
+            while bw_bytes > 0 and split_packet_count < TrafficModel.MAX_SPLIT_PACKET_COUNT:
```

Comment:
I think this means that if a stream sent more than 100 packets "at the same time", those over 100 are ignored and will not be counted by the traffic model as being sent (below, in `increment_traffic_counters()`). Is that correct?

Won't that bias our model into thinking Tor is sending less than it actually is? Because the traffic model increments counters `for i in xrange(num_packets)`.